examples/placeholder/linux/apps/app1/config.matter

```diff
@@ -2320,6 +2320,10 @@ cluster BridgedDeviceBasicInformation = 57 {
     kFabric = 5;
```

Comment:
Blocking comment on https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9499 to merge first